index.bs

```diff
@@ -159,9 +162,9 @@ Resource names starting with U+002D HYPHEN-MINUS (-) are reserved; requesting th
 ## Lock Managers ## {#lock-managers}
 <!-- ====================================================================== -->
 
-A user agent has a <dfn>lock manager</dfn> for each [=/origin=], which encapsulates the state of all [=lock-concept|locks=] and [=lock requests=] for that origin.
+A [=/user agent=] has a <dfn>lock manager</dfn> for each [=/storage shelf=], which encapsulates the state of all [=lock-concept|locks=] and [=lock requests=] associated with a [=/storage key=].
```

Comment:
Currently https://github.com/WICG/storage-buckets/blob/gh-pages/explainer.md#accessing-storage-apis-from-buckets says that we're planning to scope WebLocks to a storage bucket, rather than a storage shelf (because you filed https://github.com/WICG/storage-buckets/issues/21 suggesting we do so).

And maybe at that point we could use the storage bottle concept directly rather than needing some of these intermediate concepts to be exposed by the storage spec as well? I.e. lock manager would sort of "be" the storage bottle map...

Reply:
Oh, I did, didn't I... okay, updated. I left it as "associated with" the bottle map for `"web-locks"` but again wording suggestions welcome. Technically the set/queue could be entries in the map, but I'm not sure that's more readable.